controls/3.6/def.cf

```diff
@@ -278,11 +278,15 @@ bundle common def
       # This class is for PosgreSQL maintenance
       # pre-defined to every Sunday at 2 a.m.
       # This can be changed later on.
-
       "postgresql_full_maintenance" expression => "postgresql_maintenance_supported.Sunday.Hr02.Min00_05";
 
-      # Vacuum monitoring_mg table every day (except Sunday, when vacuum is run on entire db)
-      "postgresql_monitoring_maintenance" expression => "postgresql_maintenance_supported.!Sunday.Hr02.Min00_05";
+      # Vacuum full monitoring_mg table every day (except Sunday, when vacuum full is run on entire db)
+      # Required only by CFEngine Enterprise Hub 3.6.0 and 3.6.1
+      "postgresql_monitoring_maintenance" expression => "postgresql_maintenance_supported.!Sunday.Hr02.Min00_05.(cfengine_3_6_0|cfengine_3_6_1)";
```

Comment:
It's mentioned in the changelog: scoping down monitoring maintenance
On Fri 3 Jul 2015 at 16:44 Nick Anderson <notifications@github.com> wrote:

> In controls/3.6/def.cf
> <https://github.com/cfengine/masterfiles/pull/506#discussion_r33868402>:
>
> >        "postgresql_full_maintenance" expression => "postgresql_maintenance_supported.Sunday.Hr02.Min00_05";
> >
> > -      # Vacuum monitoring_mg table every day (except Sunday, when vacuum is run on entire db)
> > -      "postgresql_monitoring_maintenance" expression => "postgresql_maintenance_supported.!Sunday.Hr02.Min00_
>
> 05";
> > +      # Vacuum full monitoring_mg table every day (except Sunday, when vacuum full is run on entire db)
> > +      # Required only by CFEngine Enterprise Hub 3.6.0 and 3.6.1
> > +      "postgresql_monitoring_maintenance" expression => "postgresql_maintenance_supported.!Sunday.Hr02.Min00_05.(cfengine_3_6_0|cfengine_3_6_1)";
>
> I just noticed these class names changed. Can you include that in the
> changelog please?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cfengine/masterfiles/pull/506/files#r33868402>.
>